### PR TITLE
[ACR-1830] Add docs for ml-service API

### PR DESCRIPTION
**Pull request by gichan-jang**
Add docs for ml-service API.
 - ACR: ACR-1830

### CODE REVIEW

File: docs/application/native/guides/machine-learning/machine-learning-service.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ The Machine Learning Service API provides utility interfaces for both AI develop
 
 The main features of the Machine Learning Service API include the following:
 
-- Provide AI pipelines, models, and resources as ML services via machine-learning-agent (daemon).
+- Provide AI pipelines, models, and resources as ML services via deviceMLOps.MLAgent (daemon).
```
Comment (jaeyun-jung) [inline docs/application/native/guides/machine-learning/machine-learning-service.md:7]: I recommend removing keyword daemon:
`as ML services via machine learning agent.`

Comment (gichan-jang) [reply docs/application/native/guides/machine-learning/machine-learning-service.md]: Thanks. Updated as you suggested.

**Review by jaeyun-jung — APPROVED** [review]

**Review by anyj0527 — APPROVED** [review]
LGTM

**Comment by safir-srbd** [comment]
Closing and reopening PR to trigger staging rebuild

**Review by safir-srbd — APPROVED** [review]
@gichan-jang LGTM. I am approving this PR for merging.

**Pull request merged**